l10n_ar_purchase/i18n/es.po

```diff
@@ -83,3 +83,14 @@ msgstr "Pedido de compra"
 #: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.report_purchasequotation_document
 msgid "Request for Quotation"
```

Comment:
need to update manifest version to run update of the module and synchronize new translation terms?